Ginger/GingerCoreNET/RunLib/CLILib/CLIRequestAPI.cs

```diff
@@ -0,0 +1,83 @@
+﻿#region License
+/*
+Copyright © 2014-2021 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.CoreNET.RunLib.DynamicExecutionLib;
+using Ginger.Run;
+using Ginger.SolutionGeneral;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Amdocs.Ginger.CoreNET.RunLib.CLILib
+{
+    public class CLIRequestAPI : ICLI
```

Comment:
Request API is not type of CLI as we can't run it from the command line, need to change the class name or other to avoid confusion